README.md

```diff
@@ -34,6 +34,11 @@ You may use **any resource** other than each other to complete this diagnostic.
 This includes referencing talk materials, appropriate documentation, and
 searching for help online.
 
+You should be running `grunt nag` before diagnosing any bugs, since it finds
+some of the most common sources of errors. After `grunt nag` passes, you should
+run `grunt test` to run the included tests. Tests will tell you whether of not
```

Comment:
Great catch! Except `grunt` isn't present in this repository since it's a markdown diagnostic.